File: problemas/ad-hoc/2375/2375.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI {
    static void Main(string[] args) {
        int diametro = int.Parse(Console.ReadLine());

        List<int> numeros = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();
        int altura = numeros[0];
        int largura = numeros[1];
        int profundidade = numeros[2];

        if (diametro <= altura && diametro <= largura && diametro <= profundidade) 
        {
            Console.WriteLine("S");
        } else {
            Console.WriteLine("N");
        }
    }
}

File: problemas/ad-hoc/2375/2375.jl
using Printf

diametro = parse(Int, readline())

numeros = [parse(Int32, x) for x in split(readline())]
altura = numeros[1]
largura = numeros[2]
profundidade = numeros[3]

if (diametro <= altura && diametro <= largura && diametro <= profundidade)
    @printf "S\n"
else
    @printf "N\n"
end